README.md

```diff
@@ -5,3 +5,7 @@ Some description !
 ## adding a sub-heading
 
 this is a sub-heading !
+
+## Learning branching
```

Comment:
I think that this is the master price of all the lines that are ever written in programming history. what y'll say?